File: BackEnd-Siukat/models/rumah.go
package models

type Rumah struct {
	IDRumah              int    `gorm:"primaryKey;column:id_rumah" json:"id_rumah"`
	NoPeserta            string `gorm:"column:no_peserta;type:varchar(255)" json:"no_peserta"`
	StatusKepemilikan    string `gorm:"column:status_kepemilikan;type:enum('milik_sendiri','bersama_saudara','kontrak','menumpang')" json:"status_kepemilikan"`
	LuasTanah            string `gorm:"column:luas_tanah;type:varchar(255)" json:"luas_tanah"`
	LuasBangunan         string `gorm:"column:luas_bangunan;type:varchar(255)" json:"luas_bangunan"`
	StatusSertifikat     string `gorm:"column:status_sertifikat;type:enum('hak_milik','hak_guna_bangunan','tanpa_sertifikat','tanah_girik','lainnya')" json:"status_sertifikat"`
	BiayaPbb             int    `gorm:"column:biaya_pbb" json:"biaya_pbb"`
	ScanPbb              string `gorm:"column:scan_pbb;type:varchar(255)" json:"scan_pbb"`
	BiayaKontrak         int    `gorm:"column:biaya_kontrak" json:"biaya_kontrak"`
	ScanKontrak          string `gorm:"column:scan_kontrak;type:varchar(255)" json:"scan_kontrak"`
	JumlahKepalaKeluarga int    `gorm:"column:jumlah_kepala_keluarga" json:"jumlah_kepala_keluarga"`
	Atribut              string `gorm:"column:atribut;type:enum('original','sanggah')" json:"atribut"`
}

func (Rumah) TableName() string {
	return "tb_rumah"
}

File: Be-siukat-new/siukat-service-dev_snmptn_2022/models/rumah.js
var Sequelize = require('sequelize')
var db = require('../config/database')

const Rumah = db.define('rumah', {
  id_rumah:{
    type: Sequelize.INTEGER,
    primaryKey: true
  },
  no_peserta: {
    type: Sequelize.STRING
  },
  status_kepemilikan: {
    type: Sequelize.ENUM('milik_sendiri','bersama_saudara','kontrak','menumpang')
  },
  luas_tanah: {
    type: Sequelize.STRING
  },
  luas_bangunan: {
    type: Sequelize.STRING
  },
  status_sertifikat: {
    type: Sequelize.ENUM('hak_milik','hak_guna_bangunan','tanpa_sertifikat','tanah_girik','lainnya')
  },
  biaya_pbb: {
    type: Sequelize.INTEGER
  },
  scan_pbb: {
    type: Sequelize.STRING
  },
  biaya_kontrak: {
    type: Sequelize.INTEGER
  },
  scan_kontrak: {
    type: Sequelize.STRING
  },
  jumlah_kepala_keluarga: {
    type: Sequelize.INTEGER
  },
  atribut: {
    type: Sequelize.ENUM('original', 'sanggah')
  }
},
 {
  // don't add the timestamp attributes (updatedAt, createdAt)
  timestamps: false,

  // don't delete database entries but set the newly added attribute deletedAt
  // to the current date (when deletion was done). paranoid will only work if
  // timestamps are enabled
  paranoid: true,

  // don't use camelcase for automatically added attributes but underscore style
  // so updatedAt will be updated_at
  underscored: true,

  // disable the modification of tablenames; By default, sequelize will automatically
  // transform all passed model names (first parameter of define) into plural.
  // if you don't want that, set the following
  freezeTableName: true,

  // define the table's name
  tableName: 'tb_rumah'
}
);
module.exports = Rumah
